llvm/docs/ReleaseNotes.rst

```diff
@@ -344,21 +344,26 @@ Changes to the LLVM tools
 * Fixed big-endian support in llvm-symbolizer's DWARF location parser.
 * llvm-readelf now supports ``--extra-sym-info`` (``-X``) to display extra
   information (section name) when showing symbols.
-
-* ``llvm-nm`` now supports the ``--line-numbers`` (``-l``) option to use
-  debugging information to print symbols' filenames and line numbers.
+* ``llvm-readobj``/``llvm-readelf`` now supports ``--decompress``/``-z`` with
```

Comment:
There are a number of references to tool names in other notes within this file without the backticks. They should probably be normalised at some point.

Reply:
[added more backticks](https://github.com/llvm/llvm-project/pull/83751/commits/8ba9bed8210bf7bce320f589b1243f19cd7e9c6c)